### ISSUE TITLE
PROPOSAL - Multiline interface

### ISSUE THREAD

**Issue by jrevillard** [standup-raven/standup-raven#167]
**Is your feature request related to a problem? Please describe.**

Most of the time when I fill my standup, I have to copy paste what "I planned to do" the day before and put it in the what "I realy done yesterday" section. This is quite painful actually because I cannot past directly the multiline description that I have in the previous day report.

**Describe the solution you'd like**

I would like to have the possibility to enter my tasks in a multiline edit block and the standup-raven software would automatically split it line by line to fill the report.

Best,
Jerome

**Comment by harshilsharma63** [comment]
@jrevillard thanks for the feature suggestion 👍

This sounds really cool and useful. This can be done like when you paste a text, it can detect there are multiple lines in it and ask the user if they want to insert <n> entries or just one. If they choose <n> entries, Standup Raven can itself break down the clipboard text into multiple lines and insert multiple  entries.

Just curious, would you be interested in working on this?

**Comment by jrevillard** [comment]
@harshilsharma63 thanks for you interested for this feature.

What you propose seems a good idea which does not introduce ui modification. It will have to be able to deal with multiple end of line information to be cross platform (\r\n, \n). Perhaps a configuration would be usefull in order to specify a list of eol characters to be even more generic but this is perhaps too much.

Unfortunately no I don't have time at all to work on that sorry. 

Thanks a lot for this really good tool
Best 
Jérôme

**Comment by harshilsharma63** [comment]
@jrevillard good point regarding multiple new line chars. I think it can be easily handled within JavaScript itself.

> Unfortunately no I don't have time at all to work on that sorry.

No worries mate!

**Comment by cpaulik** [comment]
For me it would already be much nicer if the questions could be answered in a multiline textbox instead of the 1 line. No need for parsing anything and breaking it up.

The reason is that I often include links to issues or merge requests which makes the answer lines much too short.

Should I open a new issue for this?
